### PR TITLE
118 codecov badge

### LINKED ISSUE #118: Docs: include coverage badge on README.md
# Issue description

Add badge to the repo README.md to show the code coverage.

We need to get a [codecov](https://codecov.io/#features) account to enable this feature.

Further to R1 we can also block PRs that decrease coverage as a way to enable tests with PRs.

## Type of Issue

Please delete options that are not relevant.

- [x] New feature request
- [x] Documentation update

### PR DESCRIPTION
**Pull request by NMontanaBrown**
# Description

Travis file was modified to show the coverage of the repo using Codecov. A badge was added to the README to illustrate the changes. The badge represents the coverage provided by unit tests written so far and pushed to master.

Fixes #118 

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ python:
   - "3.7" # current default Python on Travis CI
 install:
   - pip install -e . # install deepreg package with corresponding requirements
+  - pip install pytest-cov
```
Comment (mathpluscode) [inline .travis.yml:8]: other dependencies are described in setup.py, maybe we can add these packages there?

Comment (NMontanaBrown) [reply .travis.yml:8]: This isn't strictly a dependency for dev/usage, it's just for code coverage on repo. Still want me to add?

Comment (mathpluscode) [reply .travis.yml:8]: ah ok lol but pytest-cov is already in setup

Comment (NMontanaBrown) [reply .travis.yml:8]: :facepalm: fixing now

**Pull request merged**